fractal-puppy/Engine
# Feature/chest scene

This could use a lot more polishing to be honest. But it works for now since the VS3 is so close nearby.

To test:
Load "ChestTest" scene.
Make sure the chest root has a state machine
Press play
Move the "Player" cose to the chest, it should open like this.
![Chest script + anims](https://user-images.githubusercontent.com/40542463/58442414-5dbf5180-80eb-11e9-946f-da776efbdd96.gif)

Also enabled the option to have a clip with "no animation" created. It's on the clip combo, in the state machine inspector (Not the panel, the inspector!)
